org.eclipse.jdt.core/dom/org/eclipse/jdt/internal/core/dom/rewrite/ASTRewriteAnalyzer.java

```diff
@@ -1859,6 +1859,20 @@ public boolean visit(CompilationUnit node) {
 		return false;
 	}
 
+	@Override
+	public boolean visit(ImplicitTypeDeclaration node) {
+		if (!hasChildrenChanges(node)) {
+			return doVisitUnchangedChildren(node);
+		}
+		//javaDoc
+		int pos= rewriteJavadoc(node, ImplicitTypeDeclaration.JAVADOC_PROPERTY);
+
+		int startIndent= getIndent(node.getStartPosition()) + 1;
+		int startPos= getPosAfterLeftBrace(pos);
```

Comment:
Suby, I believe there is no left brace.  Shouldn't this be the beginning of first element in Body declarations ?

Reply:
For Implicit type declaration, the value is always zero. So can I hardcoded the value.

Reply:
Calculating the start position with `node.getStartPosition();`